pkg/operator/ceph/controller/spec.go

```diff
@@ -615,6 +634,56 @@ func StoredLogAndCrashVolumeMount(varLogCephDir, varLibCephCrashDir string) []v1
 	}
 }
 
+// GenerateMDSLivenessProbeExecDaemon generates a liveness probe that makes sure mds daemon is present in fs map,
+// that it can be called, and that it returns 0
+func GenerateMDSLivenessProbeExecDaemon(daemonID, filesystemName string) *v1.Probe {
```

Comment:
If this is mds-specific, it should be in the mds code directory and not the generalized ceph controller directory.

Reply:
Updated